### ISSUE TITLE
Open GFX Base Maglev Roofs 

### ISSUE THREAD

**Issue by supermop** [OpenTTD/OpenGFX#44]
The Open GFX Maglev station roofs do not quite match the Rail and Monorail roofs in terms of level of detail or pixel art style - generally they could just use a bit more love.  I've tried to make the curved spaceframe portion more substantial by giving it some chord depth with a lower layer, and split the coloring to be silver on top and CC below.  I've also added a CC stripe to the rather plain black columns.  I added a bit more depth to the CC clerestory on the Monorail roof while I was at it.

Best,
![Station Roof 20200518](https://user-images.githubusercontent.com/65552890/82230405-ae7e1a80-98f9-11ea-92d0-4198883da3be.png)

**Comment by supermop** [comment]
Here is the infra08 spritesheet with the changes included:
[infra08.zip](https://github.com/OpenTTD/OpenGFX/files/4645306/infra08.zip)

**Comment by Andrew350** [comment]
Couple of things:

 - It looks really nice :) I hacked it into NightGFX really quick to see what it looked like in-game:

![Senfield Transport, 03_29_2090](https://user-images.githubusercontent.com/50259667/82245352-a6bd7680-98f7-11ea-85ff-fcad1ab39bad.png)

 - The station graphics are actually pulled from infra**06**.png, which threw me off for a second when I couldn't get your changes to show up initially. Not a big deal, just something to remember if you keep it like this.

 - Related to the above, previous changes to OpenGFX have created a new .png file with the new graphics instead of modifying the existing spritesheets. Many of the old spritesheets are cluttered with old, unused sprites which leads to confusion (as my example above shows). Making new ones might be better, but not necessarily required. :)

**Comment by supermop** [comment]
> Couple of things:
> 
> It looks really nice :) I hacked it into NightGFX really quick to see what it looked like in-game:
> 
> 
> 
> 
> The station graphics are actually pulled from infra06.png, which threw me off for a second when I couldn't get your changes to show up initially. Not a big deal, just something to remember if you keep it like this.
> 
> 
> Related to the above, previous changes to OpenGFX have created a new .png file with the new graphics instead of modifying the existing spritesheets. Many of the old spritesheets are cluttered with old, unused sprites which leads to confusion (as my example above shows). Making new ones might be better, but not necessarily required. :)

Presumably if I format them in a new png, I'll need to set out the sprite sizes and locations/offsets somewhere as well right?

Do you think the graphic approach is successful, or should the spaceframe members be thicker and/or denser to better fit the style of the other two stations?

**Comment by Andrew350** [comment]
> Presumably if I format them in a new png, I'll need to set out the sprite sizes and locations/offsets somewhere as well right?<

Yes, but that info can be easily copied from the OpenGFX source, since you haven't changed any dimensions or anything :)

> Do you think the graphic approach is successful, or should the spaceframe members be thicker and/or denser to better fit the style of the other two stations? <

It's supposed to be kind of futuristic and minimalist right? I'd say it looks ok but that's totally up to you :)

**Comment by supermop** [comment]
Hi all, just trying to gauge if there is any interest in this, as I might pick it up again soon-ish

**Comment by planetmaker** [comment]
I will be happy to approve
